api_app/connectors_manager/migrations/0030_connector_config_emailsender.py

```diff
@@ -0,0 +1,212 @@
+from django.db import migrations
+from django.db.models.fields.related_descriptors import (
+    ForwardManyToOneDescriptor,
+    ForwardOneToOneDescriptor,
+    ManyToManyDescriptor,
+)
+
+plugin = {
+    "python_module": {
+        "health_check_schedule": None,
+        "update_schedule": None,
+        "module": "email_sender.EmailSender",
+        "base_path": "api_app.connectors_manager.connectors",
+    },
+    "name": "EmailSender",
+    "description": "Send email",
+    "disabled": False,
+    "soft_time_limit": 60,
+    "routing_key": "default",
+    "health_check_status": True,
+    "maximum_tlp": "AMBER",
+    "run_on_failure": True,
+    "model": "connectors_manager.ConnectorConfig",
+}
+
+params = [
+    {
+        "python_module": {
+            "module": "email_sender.EmailSender",
+            "base_path": "api_app.connectors_manager.connectors",
+        },
+        "name": "subject",
+        "type": "str",
+        "description": "Email subject",
+        "is_secret": False,
+        "required": True,
+    },
+    {
+        "python_module": {
+            "module": "email_sender.EmailSender",
+            "base_path": "api_app.connectors_manager.connectors",
+        },
+        "name": "body",
+        "type": "str",
+        "description": "Email body",
+        "is_secret": False,
+        "required": True,
+    },
+    {
+        "python_module": {
+            "module": "email_sender.EmailSender",
+            "base_path": "api_app.connectors_manager.connectors",
+        },
+        "name": "sender",
+        "type": "str",
+        "description": "Email sender",
+        "is_secret": False,
+        "required": False,
+    },
+]
+
+values = [
+    {
+        "parameter": {
+            "python_module": {
+                "module": "email_sender.EmailSender",
+                "base_path": "api_app.connectors_manager.connectors",
+            },
+            "name": "subject",
+            "type": "str",
+            "description": "Email subject",
+            "is_secret": False,
+            "required": True,
+        },
+        "analyzer_config": None,
+        "connector_config": "EmailSender",
+        "visualizer_config": None,
+        "ingestor_config": None,
+        "pivot_config": None,
+        "for_organization": False,
+        "value": "Subject",
+        "updated_at": "2024-04-22T14:08:49.711495Z",
+        "owner": None,
+    },
+    {
+        "parameter": {
+            "python_module": {
+                "module": "email_sender.EmailSender",
+                "base_path": "api_app.connectors_manager.connectors",
```

Comment:
I think that we can remove this values from the migration, because we want each user to customize it before being able to run the connector